// Copyright 2018 Eryx <evorui аt gmail dοt com>, All rights reserved.

syntax = "proto3";
package hrpc;

// option optimize_for = LITE_RUNTIME;
// option cc_generic_services = true;


message ReqMeta {
  string service_name = 1;
  string method_name = 2;
  int64 req_time = 3;
  string req_id = 4;
  int32 msg_size = 5;
  int32 msg_compress_type = 6;
  string auth_access_key = 7;
  bytes auth_data = 8;
}

message RepMeta {
  int32 error_code = 1;
  string error_message = 2;
  int64 rep_time = 3;
  int32 msg_size = 4;
  int32 msg_compress_type = 5;
}
